### PR TITLE
Login Navigation PR3

### PR DESCRIPTION
**Pull request by ghost**
### Summary
* RootActivity added
* Home and SignUp Activities and Fragments added
* RootActivity decides where to go _(HomeActivity or LoginActivity)_ depending on login status
* Sign Up button launches SignUpActivity
* Clicking termns and conditions TextView launches a browser activity to show Wolox Website
* Login button _(In LoginActivity)_ and Logout button _(In HomeActivity)_ changes the login status

### Card
https://trello.com/c/jVW499Yr/18-u1-frontend-login-navegaci%C3%B3n

### Screen Recordings
termnsAndCondBrowser
rootToLoginOrHome

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -11,13 +11,26 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:theme="@style/AppTheme">
-        <activity
-            android:name=".example.ui.example.login.LoginActivity">
+
+        <activity android:name=".example.ui.example.root.RootActivity">
```
Comment (lmiotti) [inline app/src/main/AndroidManifest.xml]: Move android:name to the next line

File: app/src/main/java/ar/com/wolox/android/example/ui/example/home/HomePresenter.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+
+/**
+ *
+ */
+class HomePresenter extends BasePresenter<IHomeView> {
+
+    @Inject
+    HomePresenter() {}
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/example/home/HomePresenter.java]: Write inject all in the same line

File: app/src/main/java/ar/com/wolox/android/example/ui/example/home/IHomeView.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package ar.com.wolox.android.example.ui.example.home;
+
+/**
+ *
+ */
+public interface IHomeView {
+
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/example/home/IHomeView.java]: Add a TODO here if it is going to be implemented later

File: app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java [hunk 4]
```diff
@@ -1,23 +1,113 @@
...
 import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+import butterknife.BindView;
 
 /**
  * A simple {@link WolmoFragment} subclass.
  */
 public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
 
+    @BindView(R.id.vLoginTermsAndConditionsTextView) TextView vLoginTermsAndConditionsTextView;
+
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java]: Remove this line

File: app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java [hunk 5]
```diff
@@ -1,23 +1,113 @@
...
+    @BindView(R.id.vLoginEmailEditText) EditText vLoginEmailEditText;
+
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java]: Same as above

File: app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java [hunk 6]
```diff
@@ -1,23 +1,113 @@
...
+    @BindView(R.id.vLoginPasswordEditText) EditText vLoginPasswordEditText;
+
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java]: Same as above

File: app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java [hunk 7]
```diff
@@ -1,23 +1,113 @@
...
+    @BindView(R.id.vLoginLogInButton) Button vLoginLogInButton;
+
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java]: Same as above

File: app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginPresenter.java [hunk 8]
```diff
@@ -11,4 +12,39 @@
 
     @Inject
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginPresenter.java]: Write injects in the same line

File: app/src/main/java/ar/com/wolox/android/example/ui/example/signup/SignUpPresenter.java [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+import javax.inject.Inject;
+
+import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+
+/**
+ *
+ */
+public class SignUpPresenter extends BasePresenter<ISignUpView> {
+
+    @Inject
```
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/example/signup/SignUpPresenter.java]: Same as comment above

File: app/src/main/res/layout/fragment_sign_up.xml [hunk 10]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
```
Comment (lmiotti) [inline app/src/main/res/layout/fragment_sign_up.xml]: Move layout_width on a new line

**Review by lmiotti — CHANGES_REQUESTED** [review]
This PR is awesome. Just minor comments. Keep on working like this!

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (ghost) [reply app/src/main/AndroidManifest.xml]: ✔ **Done**

File: app/src/main/java/ar/com/wolox/android/example/ui/example/home/HomePresenter.java (hunk 2 above)
Comment (ghost) [reply app/src/main/java/ar/com/wolox/android/example/ui/example/home/HomePresenter.java]: ✔ **Done**

File: app/src/main/java/ar/com/wolox/android/example/ui/example/home/IHomeView.java (hunk 3 above)
Comment (ghost) [reply app/src/main/java/ar/com/wolox/android/example/ui/example/home/IHomeView.java]: ✔ **Done**

File: app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java (hunk 4 above)
Comment (ghost) [reply app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java]: ✔ **Done**

File: app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java (hunk 5 above)
Comment (ghost) [reply app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java]: ✔ **Done**

File: app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java (hunk 6 above)
Comment (ghost) [reply app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java]: ✔ **Done**

File: app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java (hunk 7 above)
Comment (ghost) [reply app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java]: ✔ **Done**

File: app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginPresenter.java (hunk 8 above)
Comment (ghost) [reply app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginPresenter.java]: ✔ **Done**

File: app/src/main/java/ar/com/wolox/android/example/ui/example/signup/SignUpPresenter.java (hunk 9 above)
Comment (ghost) [reply app/src/main/java/ar/com/wolox/android/example/ui/example/signup/SignUpPresenter.java]: ✔ **Done**

File: app/src/main/res/layout/fragment_sign_up.xml (hunk 10 above)
Comment (ghost) [reply app/src/main/res/layout/fragment_sign_up.xml]: ✔ **Done**

**Comment by ghost** [comment]
> This PR is awesome. Just minor comments. Keep on working like this!

Requested changes done! :+1:

**Review by lmiotti — APPROVED** [review]
di_caprio

**Comment by lmiotti** [comment]
Be cafeful before merging, there some conflicts to be resolved before

**Pull request merged**